Yoast/wordpress-seo
# Pass results field instead of object to SeoAnalysis

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* In the feature-branch, there was some future-proofing by shifting the analysis results into an object, into the `results` field. This was done to later have the ability to put an `overallScore` field for each keyword, that could be used to color a collapsible sidebar icon in accordance with the analysis score. Somewhere something went wrong in the sense that this deeper structure was not connected to `SeoAnalysis` and `ReadabilityAnalysis` properly. This is now fixed.

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and link yoast components
* Verify that all analysis results are coming in both free and premium. To test the code in premium, see https://github.com/Yoast/Wiki/wiki/Merging-Free-into-Premium . **Do not commit to premium though!**

## Quality assurance


Requires https://github.com/Yoast/yoast-components/pull/649
